user-manager/src/main/java/com/qpay/usermanager/utils/PathsUtils.java

```diff
@@ -0,0 +1,9 @@
+package com.qpay.usermanager.utils;
+
+public final class PathsUtils {
```

Comment:
https://projectlombok.org/features/experimental/UtilityClass
can help to reduce boilerplate code